src/components/CommonUserModal.tsx

```diff
@@ -335,7 +375,7 @@ const AddLearnerModal: React.FC<AddLearnerModalProps> = ({
               handleBlockChangeWrapper={handleBlockChangeWrapper}
               isMobile={isMobile}
               isMediumScreen={isMediumScreen}
-              isCenterSelection={true}
+              isCenterSelection={userType!=="TEAM LEADER"?true: false}
```

Comment:
@AkshataKatwal16 take it from the app.constant

Reply:
done